app/assets/stylesheets/fullcalendar_engine/style.css

```diff
@@ -17,7 +12,3 @@ body{
 span > div#loading {
   position: fixed;
 }
-
-h1{
-font-size: 16px;
-}
```

Comment:
We can't do this straight away, as this may be possible that someone else is using the default css.